input/pagecontent/StructureDefinition-mcode-cancer-patient-intro.md

```diff
@@ -1,3 +1,4 @@
 ### Conformance
 
-Patient resources representing mCODE patients (as defined on the [Identifying mCODE Patients](conformance-patients.html) page) MUST conform to this profile. Any resource intended to conform to this profile SHOULD populate `meta.profile` accordingly.
+Patient resources representing Cancer Patients in the scope of mCODE (as defined on the [Identifying mCODE Patients](conformance-patients.html) page) MUST conform to this profile. Any resource intended to conform to this profile SHOULD populate `meta.profile` accordingly.
```

Comment:
```suggestion
Patient resources representing Cancer Patients in the scope of mCODE (as defined on the [Identifying in-scope Patients](conformance-patients.html) page) MUST conform to this profile. Any resource intended to conform to this profile SHOULD populate `meta.profile` accordingly.
```

Reply:
Fix Capitalization: "Identifying In-Scope Patients".